content/07.pathogenesis.md

```diff
@@ -148,16 +147,16 @@ Cytokines have been investigated for their role in the immune response to lung i
 Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with Acute Respiratory Distress Syndrome (ARDS), which is a severe condition that can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104].
```

Comment:
I think you abbreviate ARDS in intro section

Reply:
```suggestion
Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with ARDS, which is a severe condition that can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104].
```